supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc

```diff
@@ -318,6 +318,17 @@ The API object for a deployment can be either a Kubernetes-native `Deployment` o
 
 *See also*:
 
+discrete]
+[[devfile]]
+==== image:images/yes.png[yes] Devfile (noun)
+*Description*: A _devfile_ is a YAML file that you can include in your local development environment to automate various build processes and apply environment guidelines. Specifically, devfiles can provide guidance for runtime images, example code, build and CI commands, and deployment options like workspace servers and containerization.
```

Comment:
```suggestion
*Description*: A _devfile_ is a YAML file that you can include in your local development environment to automate various build processes and apply environment guidelines. Specifically, devfiles can provide guidance for runtime images, example code, build and CI commands, and deployment options such as workspace servers and containerization.
```

Reply:
@jseseCCS Ping me once you have revised the PR. I will approve it from my end.

Reply:
thanks! it needs a SME review, no? i've been pinging the SME my teammates IDd but no response so far.

Reply:
Np, good luck with it.

Reply:
Just to confirm that the guidelines are applied to the build environment or an application? I feel that "and apply environment guidelines" dangles a bit, but it could be me!

Reply:
thanks for your review, @reginapizza! no it's correct as i wrote it: CI commands

Reply:
ah I was reading it wrong, I understand what you're saying now. Nvm, looks good then :)